test/measures/measures-data-spec.js

```diff
@@ -31,5 +31,40 @@ describe('measures data json', function() {
 
       assert.equal(intersection.size, multiPerformanceIds.size);
     });
+
+    describe('CAHPS measures', function() {
+      const cahpsMeasures = measuresData.filter(function(measure) {
+        var re = /CAHPS_\d+/i;
+        return measure.measureId.match(re) !== null;
+      });
```

Comment:
I think your right the function could be simpler but I thought arrow functions were discouraged? But updating the function callback to just `measure.measureId.match(/CAHPS_\d+/));`